src/kirby/components/icon/icon.component.scss

```diff
@@ -4,28 +4,34 @@
   display: inline-block;
   font-size: size('m');
 
+  .outline {
+    border: solid 3px currentColor;
+    padding: size('xs');
+    border-radius: calc(#{size('xxxl')} * 2);
```

Comment:
Consider removing `calc` as it might as well be done compile time

Reply:
If calc should be done at compile time we need to adjust our size function, but I agree that it would be preferable.